specification/e-referrals-service-api.yaml

```diff
@@ -3509,10 +3514,87 @@ paths:
           $ref: 'components/schemas/responses/cancelReferral/502Response.yaml'
         '504':
           $ref: 'components/schemas/responses/cancelReferral/504Response.yaml'   
+  /STU3/CommunicationRequest/{ubrn}/$ers.generateCRI:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to generate a PDF file that summarises an advice and guidance request including the conversation. 
+        This can only be be used for a UBRN which is an advice and guidance request that has *not* been converted to a referral.
```

Comment:
"be" repeated twice on the line 3535 i.e. "**This can only be be**"

Reply:
Good spot. I'll remove